src/OSPSuite.Core/Domain/QuantityPKParameterContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using OSPSuite.Core.Domain.UnitSystem;
+
+namespace OSPSuite.Core.Domain
+{
+   public class QuantityPKParameterContext: IWithDimension
```

Comment:
The context is a trick we use to be able to create a converter for the QuantityPKPArameter